src/config/scrivitoContentBrowser.js

```diff
@@ -1,107 +1,129 @@
-          field: "_objClass",
-          value: "Image",
-        },
-        Pages: {
-          title: "Pages",
-          icon: "sheet",
-          field: "_objClass",
-          value: [
-            "Author",
-            "Blog",
-            "BlogPost",
-            "Event",
-            "Homepage",
-            "Job",
-            "LandingPage",
-            "Page",
-            "Redirect",
-            "SearchResults",
-          ],
-          options: {
-            Page: {
-              title: "Standard pages",
-              icon: "sheet",
-            },
-            LandingPage: {
-              title: "Landing pages",
-              icon: "inbox",
-            },
-            BlogPost: {
-              title: "Blog posts",
-              icon: "pen",
-            },
-            Author: {
-              title: "Authors",
-              icon: "user",
-            },
-            Event: {
-              title: "Events",
-              icon: "cal",
-            },
-            Job: {
-              title: "Jobs",
-              icon: "suitcase",
-            },
-            Redirect: {
-              title: "Redirects",
-              icon: "link",
-            },
-            Homepage: {
-              title: "Homepage",
-              icon: "inbox",
-            },
-            Blog: {
-              title: "Blog",
-              icon: "pen",
+  filters: filterContext => {
+    if (filterContext._validObjClasses && filterContext._validObjClasses[0]) {
+      const validObjClasses = filterContext._validObjClasses;
+      const [head, ...tail] = validObjClasses;
+
+      if (tail.length) {
+        return {
+          _objClass: {
+            options: {
+              All: {
+                title: "All",
+                icon: "folder",
+                field: "_objClass",
+                value: validObjClasses,
+              },
+              [head]: {
+                ...descriptionForObjClass(head),
+                selected: true,
+              },
+              ...tail.reduce((interimOptions, item) => {
+                return {
+                  ...interimOptions,
+                  ...{ [item]: descriptionForObjClass(item) },
+                };
+              }, {}),
             },
-            SearchResults: {
-              title: "Search results",
-              icon: "lens",
+          },
+          _modification: modificationFilter,
+        };
+      }
+
+      return {
+        _objClass: {
+          options: {
+            [head]: {
+              ...descriptionForObjClass(head),
+              selected: true,
             },
           },
         },
-        Download: {
-          title: "Downloads",
-          icon: "pdf",
-          field: "_objClass",
-          value: "Download",
-        },
-        Video: {
-          title: "Videos",
-          icon: "video",
-          field: "_objClass",
-          value: "Video",
+        _modification: modificationFilter,
+      };
+    }
+
+    const pages = [
```

Comment:
Extract all code into a (clearly non-parametrized) `return defaultFiltersDefinition()` for better code climate?

AFAICS it is even a constant object - not even `new Date()` involved - similar to `_modificationFilter`.